components/Landing/HeroCarousel.tsx

```diff
@@ -3,13 +3,16 @@ import NextImage from 'next/image'
 
 import { styled } from '@config/stitches'
 
-import Img1 from '@assets/carousel-image1.jpg'
+import Img1 from '@assets/carousel-img1.jpg'
+import Img2 from '@assets/carousel-img2.jpg'
+import Img3 from '@assets/carousel-img3.jpg'
+
```

Comment:
```suggestion
import Img1 from '@assets/carousel-img1.jpeg'
import Img2 from '@assets/carousel-img2.jpeg'
import Img3 from '@assets/carousel-img3.jpeg'
```

![image](https://user-images.githubusercontent.com/28263450/140486568-bcf913fd-a3f6-48b3-a4e5-50a60d7aa4e8.png)

also please ko remove ani na file `public/assets/carousel-image1.jpg`